packages/Toast/stories/examples/UncontrolledToastExample.js

```diff
@@ -1,15 +1,30 @@
 import React from "react";
 
+import CodeViewer from "storybook/components/CodeViewer";
 import L10n from "@paprika/l10n";
 import Toast from "../../src";
 
 export default function UncontrolledToast() {
+  const [accumulator, setAccumulator] = React.useState(0);
+
+  function handleItem() {
+    setAccumulator(prev => prev + 1);
+  }
+
   return (
     <L10n locale="en">
       <Toast hasCloseButton>Uncontrolled toast component</Toast>
       <Toast canAutoClose autoCloseDelay={5000} hasCloseButton>
         Uncontrolled toast component with auto close
       </Toast>
+      <CodeViewer>
```

Comment:
While I appreciate the creative use of the `<CodeViewer>` here, I think maybe it would be more useful to just link to the whole story code on GitHub – a pattern established in several other components stories (`<Input>` is one example you can reference).  Essentially by using the `<ExampleStory>` helper we can include a `source` link in the header.